x-pack/platform/plugins/shared/features/server/feature_privilege_iterator/feature_privilege_iterator.test.ts

```diff
@@ -822,6 +822,7 @@ describe('featurePrivilegeIterator', () => {
             settings: ['cases-settings-type', 'cases-settings-sub-type'],
             createComment: ['cases-create-comment-type', 'cases-create-comment-sub-type'],
             reopenCase: ['cases-reopen-type', 'cases-reopen-sub-type'],
+            assign: [],
```

Comment:
Question: why are the `assign` arrays the only empty arrays on these 3 tests?

Reply:
I think because having exact copies of current permissions are not really needed to test most of what's being tested in this file, but I went ahead and added them in https://github.com/elastic/kibana/pull/201654/commits/e9d4f337a3ff106fe84a0f3157a73425b93eab2c

Reply:
thanks!